tcp-ballerina/build.gradle

```diff
@@ -296,6 +294,13 @@ publishing {
     }
 }
 
-build {
-    dependsOn(ballerinaBuild)
-}
+ballerinaBuild.dependsOn copyToLib
```

Comment:
Since we have moved all the task dependencies of the` ballerinaBuild` task here, shall we move the dependencies of `ballerinaTest` task here too?

You can refer [this gradle file](https://github.com/ballerina-platform/module-ballerina-graphql/blob/f2646f8d71c3b7154e60c3f2e21463300fcd59aa/graphql-ballerina/build.gradle#L259)

Reply:
I will do that

Reply:
I did the changes